content/hi/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -21,66 +21,66 @@ sitemap:
 
-वर्चुअलाइजेशन एक भौतिक सर्वर में संसाधनों के बेहतर उपयोग की अनुमति देता है और बेहतर मापनीयता की अनुमति देता है क्योंकि एक एप्लिकेशन को आसानी से जोड़ा या अपडेट किया जा सकता है, हार्डवेयर लागत को कम करता है, और बहुत कुछ। वर्चुअलाइजेशन के साथ आप भौतिक संसाधनों का एक सेट डिस्पोजेबल वर्चुअल मशीनों के समूह के रूप में प्रस्तुत कर सकते हैं।
+वर्चुअलाइजेशन एक भौतिक सर्वर में संसाधनों के बेहतर उपयोजन की अनुमति देता है और बेहतर मापक्रमणीयता की अनुमति देता है क्योंकि एक एप्लिकेशन को आसानी से जोड़ा या नवीनीकरण किया जा सकता है, यंत्रसामग्री के लागत को कम करता है, और बहुत कुछ। वर्चुअलाइजेशन के साथ आप भौतिक संसाधनों का एक सेट प्रयोगशील वर्चुअल मशीनों के समूह के रूप में प्रस्तुत कर सकते हैं।
 
-प्रत्येक वीएम वर्चुअलाइज्ड हार्डवेयर के शीर्ष पर अपने स्वयं के ऑपरेटिंग सिस्टम सहित सभी घटकों को चलाने वाली एक पूर्ण मशीन है।
+प्रत्येक वर्चुअल मशीन एक पूरी मशीन की तरह होती है, जिसमें अपने ऑपरेटिंग सिस्टम के साथ सभी घटक भी होते हैं, और ये सब वर्चुअलाइज्ड हार्डवेयर पर चलते हैं।
 
-**कंटेनर डिप्लॉयमेंट युग:** कंटेनर VMs के समान होते हैं, लेकिन उनके पास एप्लिकेशनो के बीच ऑपरेटिंग सिस्टम (OS) को साझा करने के लिए अलगाव गुण होते हैं। इसलिए, कंटेनरों को हल्का माना जाता है। वीएम (VM) के समान, एक कंटेनर का अपना फाइल सिस्टम, सीपीयू का हिस्सा, मेमोरी, प्रोसेस स्पेस और बहुत कुछ होता है। चूंकि वे अंतर्निहित बुनियादी ढांचे से अलग हो गए हैं, वे बादलों और ओएस (OS) वितरण में पोर्टेबल हैं।
+**कंटेनर डिप्लॉयमेंट युग:** कंटेनर VMs के समान होते हैं, लेकिन उनके पास एप्लिकेशनो के बीच ऑपरेटिंग सिस्टम (OS) को साझा करने के लिए एकाकीपन की स्वतंत्रता होती है। इसलिए, कंटेनरों को हल्का माना जाता है। वीएम (VM) के समान, एक कंटेनर का अपना फाइल सिस्टम, सीपीयू का हिस्सा, मेमोरी, प्रोसेस स्पेस और बहुत कुछ होता है। चूंकि वे अंतर्निहित अवसंरचना से अलग हो गए हैं, वे बादलों और ओएस (OS) वितरण में सुवाह्य  हैं।
 
 कंटेनर लोकप्रिय हो गए हैं क्योंकि वे अतिरिक्त लाभ प्रदान करते हैं, जैसे:
 
-* अजाइल (Agile) एप्लिकेशन निर्माण और डिप्लॉयमेंट: VM इमेज उपयोग की तुलना में कंटेनर इमेज निर्माण की आसानी और दक्षता में वृद्धि हुई है।
-* निरंतर विकास, एकीकरण और डिप्लॉयमेंट: विश्वसनीय और लगातार कंटेनर इमेज निर्माण और त्वरित और कुशल रोलबैक के साथ तैनाती (इमेज अपरिवर्तनीयता के कारण) प्रदान करता है।
-* डेव (Dev) और ऑप्स (Ops) चिंताओं का पृथक्करण: डिप्लॉयमेंट समय के बजाय बिल्ड/रिलीज़ समय पर एप्लिकेशन कंटेनर इमेजेस (images) बनाएं, जिससे बुनियादी ढांचे से एप्लिकेशनो को अलग किया जा सके।
+* अजाइल (Agile) एप्लिकेशन निर्माण और डिप्लॉयमेंट: VM इमेज उपयोग की तुलना में कंटेनर इमेज निर्माण की आसानी और कार्यक्षमता में वृद्धि हुई है।
+* निरंतर विकास, एकीकरण और डिप्लॉयमेंट: विश्वसनीय और नियमित कंटेनर इमेज निर्माण और त्वरित और कुशल रोलबैक के साथ तैनाती (इमेज अपरिवर्तनीयता के कारण) प्रदान करता है।
+* डेव (Dev) और ऑप्स (Ops) चिंताओं का पृथक्करण: डिप्लॉयमेंट समय के बजाय बिल्ड/रिलीज़ समय पर एप्लिकेशन कंटेनर इमेजेस (images) बनाएं, जिससे अवसंरचना से एप्लिकेशनो को अलग किया जा सके।
```

Comment:
```suggestion
* Dev और Ops चिंताओं का पृथक्करण: डिप्लॉयमेंट समय के बजाय बिल्ड/रिलीज़ समय पर एप्लिकेशन कंटेनर इमेजिस  बनाएं, जिससे बुनियादी ढांचे से एप्लिकेशनो को अलग किया जा सके।
```

Reply:
बुनियादी ढांचे - is about structure.
अवसंरचना - is about infrastructure.